philosopher/2.0.0/Dockerfile

```diff
@@ -0,0 +1,28 @@
+################## BASE IMAGE ######################
+FROM biocontainers/biocontainers:debian-stretch-backports
+
+################## METADATA ######################
+LABEL base_image="biocontainers/biocontainers:debian-stretch-backports"
+LABEL version="2.0.0"
+LABEL software="philosopher"
+LABEL software.version="2.0.0"
+LABEL about.summary="A complete toolkit for shotgun proteomics data analysis"
+LABEL about.home="https://philosopher.nesvilab.org"
+LABEL about.documentation="https://github.com/Nesvilab/philosopher/wiki"
+LABEL about.license_file="https://github.com/Nesvilab/philosopher"
+LABEL about.license="SPDX:GPL-3.0"
+LABEL about.tags="Proteomics"
+
+################## MAINTAINER ######################
+MAINTAINER Felipe da Veiga Leprevost <felipe@leprevost.com.br>
+
+USER root
+RUN apt-get update && apt-get install -y wget
+RUN ZIP=philosopher_linux_amd64.deb && \
+  wget https://github.com/Nesvilab/philosopher/releases/download/v2.0.0/$ZIP -O /tmp/$ZIP && dpkg -i /tmp/$ZIP
+RUN rm -rfv /tmp/$ZIP
```

Comment:
You should put the rm on previous line, this will lower container size.

Reply:
I just did the changes. Please have look.